packages/edit-site/src/components/template-actions/index.js

```diff
@@ -11,12 +11,17 @@ import { store as noticesStore } from '@wordpress/notices';
 /**
  * Internal dependencies
  */
-import { store as editSiteStore } from '../../../store';
-import isTemplateRemovable from '../../../utils/is-template-removable';
-import isTemplateRevertable from '../../../utils/is-template-revertable';
+import { store as editSiteStore } from '../../store';
+import isTemplateRemovable from '../../utils/is-template-removable';
+import isTemplateRevertable from '../../utils/is-template-revertable';
 import RenameMenuItem from './rename-menu-item';
 
-export default function Actions( { template } ) {
+export default function TemplateActions( {
```

Comment:
It is great that you were able to reuse this. Personally I always prefer reusable components that receive "ids" instead of objects like this as it's more reusable and also more performant in general (can add extra selectors but this won't force the parent to re-render if it's only this component that cares about the object)

Reply:
I updated to fetch the data in this component. Also used `getEntityRecord` and reverted the checks for `title.rendered` existence.